packages/python-runner/src/widget.tsx

```diff
@@ -214,9 +214,24 @@ export class PythonFileEditor extends DocumentWidget<
 
     if (this.dockPanel.isEmpty) {
       // Add a tab to dockPanel
-      this.scrollingWidget = new ScrollingWidget({
+      const scrollingWidget = new ScrollingWidget({
         content: this.outputAreaWidget
       });
+      this.scrollingWidget = scrollingWidget;
+      const scrollUpButton = document.createElement('button');
+      const scrollDownButton = document.createElement('button');
+      scrollUpButton.className = 'elyra-PythonEditor-scrollTop';
+      scrollUpButton.innerHTML = '↑';
```

Comment:
we shouldn't use unicode characters directly in code
```suggestion
      scrollUpButton.innerHTML = '&uarr;';
```